manifests/init.pp

```diff
@@ -5,7 +5,7 @@
 # $admin_username::             Username for the initial admin user
 #                               type:String
 #
-# $admin_password::             Password of the initial admin user, default is randomly generated
```

Comment:
I think this is still valuable so users know that it's just for their installation.